XamlControlsGallery/SettingsPage.xaml

```diff
@@ -51,7 +51,11 @@
                     FontWeight="Normal"
                     Text="Sound" />
                 <StackPanel Margin="0,10,0,0">
-                    <ToggleSwitch x:Name="soundToggle" Toggled="soundToggle_Toggled" OffContent="Off" OnContent="On"/>
+                    <ToggleSwitch x:Name="soundToggle" Header="Sound" Toggled="soundToggle_Toggled" OffContent="Off" OnContent="On">
+                        <ToggleSwitch.HeaderTemplate>
+                            <DataTemplate><!-- empty template to support correct visuals and Narrator experience --></DataTemplate>
```

Comment:
This is strange. Since the ToggleSwitch has a Header set, shouldn't it do the "right thing" by default? I'm concerned that this might be an indication that ToggleSwitch controls don't have the right Narrator experience by default.

Reply:
It didn't have a header set. The "Sound" text is a distinct TextBlock. I added a header, which got me the correct Narrator behavior but resulted in two "sound" headers. So I'm heading the ToggleSwitch's built-in header to keep the visuals consistent with the current, larger text size, implementation.